ci/content/vocab.txt

```diff
@@ -1,11 +1,27 @@
 AMQP
+argparse
+Armin
+async
+asyncio
 Autoscaler
 autoscaling
+awaitable
+Backends
 barebones
+BigQuery
 ceps
+Config
+coroutine
+coroutines
+cryptographically
+deduplication
```

Comment:
minor quip, but of course this word would make this list twice.

Reply:
The vocabulary is case sensitive.